chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_lvm.py

```diff
@@ -39,7 +39,7 @@ def device_path(self):
 
     @classmethod
     def clear_device_commands(cls, device_paths):
-        return ["if vgdisplay %s; then vgremove -f %s; else exit 0; fi" % (TestBlockDeviceLvm('lvm', device_path).vg_name,
+        return ["if vgdisplay %s; then vgremove %s; else exit 0; fi" % (TestBlockDeviceLvm('lvm', device_path).vg_name,
```

Comment:
I think removing the `-f` from `vgremove` will make it want to prompt for confirmation.

Reply:
It did, but it appeared to mention there were still some lv's.

I added a lvremove before that, so that should remove the confirmation.